src/ipaperftest/core/constants.py

```diff
@@ -165,8 +165,9 @@ def getLevelName(level):
 ANSIBLE_FETCH_FILES_PLAYBOOK = """
 ---
 - name: Fetch IPA server log files
-  hosts: ipaserver, ipareplicas
+  hosts: ipaserver, ipareplicas*
   become: yes
+  ignore_errors: yes
```

Comment:
Why is ignore_errors necessary?

Reply:
This playbook is applied to both the original server and replicas. The server doesn't have `ipareplica-install.log`, and replicas don't have `ipaserver-install.log`, so the playbook would always fail when fetching files from the list without this.

Reply:
I'm a little concerned about ignoring errors but this is just pulling logs so may not be critical. We can re-address in the future if it is an issue.